x-pack/plugins/cases/public/components/custom_fields/text/configure.test.tsx

```diff
@@ -30,24 +30,36 @@ describe('Configure ', () => {
     expect(screen.getByText(i18n.FIELD_OPTION_REQUIRED)).toBeInTheDocument();
   });
 
-  it('updates field options correctly', async () => {
+  it('updates field options correctly when not required', async () => {
     render(
       <FormTestComponent onSubmit={onSubmit}>
         <Configure />
       </FormTestComponent>
     );
 
-    userEvent.click(screen.getByText(i18n.FIELD_OPTION_REQUIRED));
+    userEvent.click(screen.getByText('Submit'));
+
+    await waitFor(() => {
+      // data, isValid
+      expect(onSubmit).toBeCalledWith({}, true);
+    });
+  });
 
+  it('updates field options correctly when required', async () => {
```

Comment:
Can you add a test where you fill out the `defaultValue`?

Reply:
Good idea 👍 

Additionally, I will:
- replace the `getBy*`s in this file by `findBy*`s
- change the instances of `ByText` to `byTestId` here and in  `/text/configure.test.tsx`.